.github/workflows/PR-content-check.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: PR Content Check
+
+on: [pull_request]
+
+jobs:
+  check-PR-content:
+    name: Check PR Content
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the incoming pull request
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+          path: ion-java-new
+
+      - name: Check the content of the last commit
+        id: check-content
+        run: |
+          cd ion-java-new
+          if [[ $(git log -1 --name-only) == *"src/"* ]]; then echo "result=pass"; else exit 1; fi
```

Comment:
Is failing the workflow the only way to achieve the behavior we want? Ideally this job would always succeed, and we'd simply convey the result to the other jobs so to determine if they should run.

Reply:
This not the only way. We can add the steps of PR-content-check to both ion-java-regression-detection.yml and ion-test-driver.yml. Since both of the share the same logic of checking PR content, I'd prefer to reuse the workflow PR-content-check.yml in both applicable workflows to avoid duplication.